mne/channels/interpolation.py

```diff
@@ -213,3 +213,62 @@ def _interpolate_bads_meg(inst, mode='accurate', origin=(0., 0., 0.04),
     info_to = pick_info(inst.info, picks_bad)
     mapping = _map_meg_channels(info_from, info_to, mode=mode, origin=origin)
     _do_interp_dots(inst, mapping, picks_good, picks_bad)
+
+
+@verbose
+def _interpolate_bads_nirs(inst, method='nearest', verbose=None):
```

Comment:
This should be added to `InterpolateMixin.interpolate_bads` in `mne/channels/channels.py`. You can add a `fnirs_method` kwarg to the function.